python_ta/checkers/forbidden_import_checker.py

```diff
@@ -119,3 +132,13 @@ def get_allowed_local_files(self) -> list:
 def register(linter: PyLinter) -> None:
     """Required method to auto register this checker"""
     linter.register_checker(ForbiddenImportChecker(linter))
+
+
+def _get_full_import_names(modname: str, names: list) -> list:
+    """Given a module name and a list of functions imported from the module, return a list of strings
```

Comment:
I also realized when reviewing this that you can import all types of variables---not just functions---from modules. So a better description of this task, that should be reflected in all naming/documentation, is "allowed importing specific names for ForbiddenImportChecker". Similarly here say "list of functions" instead of "list of names"